src/main/java/nl/thehyve/ocdu/models/OcDefinitions/SiteDefinition.java

```diff
@@ -20,7 +20,12 @@
 
     private boolean isGenderRequired = true;
 
-    private boolean isBirthdateRequired = true;
+    /*
+     * 1. yes, required
+     * 2. only year of birth
```

Comment:
Are you sure it is allowed to register patient with full date (more than only year)|?